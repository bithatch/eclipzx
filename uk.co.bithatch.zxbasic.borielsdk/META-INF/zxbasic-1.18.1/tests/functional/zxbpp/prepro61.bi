#define x(b) xx b a(b)

#define a(b) aa b x(b)

x(1)
