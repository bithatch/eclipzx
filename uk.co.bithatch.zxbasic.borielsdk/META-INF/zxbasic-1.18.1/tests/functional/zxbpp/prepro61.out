#line 1 "prepro61.bi"


#line 4 "prepro61.bi"

xx 1 aa 1 x(1)
